### PR TITLE
DYN-7201 Fix info bubble updates

### PR DESCRIPTION
**Pull request by pinzart90**
Issue:
Info bubbles (as opposed to warnings or errors) do not show up on nodes after graph execution
image

### Purpose

In this PR I updated the node GUIDS that are passed to the `UpdateNodeInfoBubbleContent` method

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### Release Notes

Fix regression where information bubbles did not show on nodes after graph execution

### Reviewers

(FILL ME IN) Reviewer 1 (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs [hunk 1]
```diff
@@ -708,6 +708,9 @@ private void OnUpdateGraphCompleted(AsyncTask task)
                 }
             }
 
+            // All nodes that have runtime warnings or infos
+            HashSet<Guid> nodesWithInfos = [.. warnings.Keys];
```
Comment (aparajit-pratap) [inline src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs:712]: Can you explain this change? It looks like the guids need to be unique? What if a node has both a warning and info message - won't there be repeated guids in that case?

File: src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs [hunk 2]
```diff
@@ -716,6 +719,8 @@ private void OnUpdateGraphCompleted(AsyncTask task)
                 if (node == null)
                     continue;
 
+                nodesWithInfos.Add(guid);
```
Comment (aparajit-pratap) [inline src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs:722]: How was it working before without adding this guid here?

File: src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs (hunk 1 above)
Comment (pinzart90) [reply src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs:712]: these node guids are used to decide if we need to update the bubble info model and UI after a node has been executed. 
The guid represents nodes (not warnings/infos)
https://github.com/DynamoDS/Dynamo/blob/7a58fa79a95bd7850b114c4d6cb51218a2b6975e/src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs#L355-L366

File: src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs (hunk 2 above)
Comment (pinzart90) [reply src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs:722]: It was working before I broke it by adding the boolean `BlockInfoBubbleUpdates`
```
                using (Disposable.Create(() => { node.BlockInfoBubbleUpdates = true; }, () => { node.BlockInfoBubbleUpdates = false; }))
                {
                    node.Info(string.Join(Environment.NewLine, info.Value.Select(w => w.Message)));
                }
```
All calls to ` node.Info` used to trigger an UpdateBubbleInfo directly. But with the `BlockInfoBubbleUpdates` we stop UpdateBubbleInfo
```
        public void UpdateBubbleContent()
        {
            if (NodeModel.BlockInfoBubbleUpdates)
            {
                return;
            }

```

Comment (pinzart90) [reply src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs:722]: In this PR, by adding the node guids to `nodesWithInfos` we ensure that all affected nodes will call UpdateBubbleContent only one time at the end of the graph execution

**Review by aparajit-pratap — APPROVED** [review]

**Pull request merged**